src/core/providers/qgsabstractdatabaseproviderconnection.h

```diff
@@ -533,6 +533,7 @@ class CORE_EXPORT QgsAbstractDatabaseProviderConnection : public QgsAbstractProv
       SinglePoint = 1 << 5,            //!< Supports single point types (as distinct from multi point types) \since QGIS 3.28
       SingleLineString = 1 << 6,       //!< Supports single linestring types (as distinct from multi line types) \since QGIS 3.28
       SinglePolygon = 1 << 7,          //!< Supports single polygon types (as distinct from multi polygon types) \since QGIS 3.28
+      Surfaces = 1 << 8,              //!< Supports surface (PolyhedralSurface, TIN) types (as distinct from multi polygon types) \since QGIS 3.40
```

Comment:
Surfaces also encompasses Polygon or CurvePolygon, so may be bit more specific with:
```suggestion
      PolyhedralSurfaces = 1 << 8,              //!< Supports surface (PolyhedralSurface, TIN) types (as distinct from multi polygon types) \since QGIS 3.40
```

Reply:
Done.